src/index.html

```diff
@@ -5,8 +5,425 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-content">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Фото
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page-content__catalog catalog">
+      <div class="catalog__card card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        >
+
+        <h2 class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__code">
+          Код товара: 195434
+        </h3>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <h4 class="card__comments">
+            Отзывов: 5
+          </h4>
+        </div>
+
+        <div class="card__price">
+          <h4 class="card__label">
+            Цена:
+          </h4>
+
+          <h4 class="card__number">
+            69 999 грн
+          </h4>
```

Comment:
Don't use `h` tags everywhere, you have also `p` 🙂
Find what is primary info and use for them `h`